omnibus/config/projects/agent.rb

```diff
@@ -21,10 +21,9 @@
   maintainer 'Datadog Packages <package@datadoghq.com>'
 end
 
-build_version do
-  source :git
-  output_format :dd_agent_format
-end
+# build_version is computed by an invoke command/function.
+# We can't call it directly from there, we pass it through the environment instead.
+build_version ENV['PACKAGE_VERSION']
```

Comment:
the variable set in the .gitlabci.yaml above is `VERSION`, but the env variable retrieved is `PACKAGE_VERSION`.  Is this right?

Reply:
ok; I see `PACKAGE_VERSION` set in the invoke task(s) below; why the two different vars?

Reply:
`VERSION` in the .gitlab-ci.yaml was there before and is used for a completely different thing. I chose `PACKAGE_VERSION` because i was thinking `VERSION` was too generic and could conflict with something else